src/sage/graphs/generators/families.py

```diff
@@ -787,14 +786,14 @@ def LollipopGraph(n1, n2):
 
 def TadpoleGraph(n1, n2):
     r"""
-    Return a tadpole graph with n1+n2 nodes.
+    Return a tadpole graph with ``n1+n2`` nodes.
 
```

Comment:
I think it is okay to write
```
`n_1 + n_2`
```
as math. Also below.

Reply:
I change that in several places.